src/erik_rullestad_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,18 @@
+def bubble_sort(data):
+    len_data = len(data) - 1
```

Comment:
Please name it correctly. `len(data) - 1` doesn't mean `len_data`. It causes confusion.